src/Builders/FilterBuilder.php

```diff
@@ -125,6 +125,46 @@ public function when($value, callable $callback, callable $default = null)
         return $this;
     }
 
+    /**
+     * https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-distance-query.html
+     * @param string $field
+     * @param string $value
+     * @param array|string $distance
+     * @return $this
+     */
+    public function whereGeoDistance($field, $value, $distance)
```

Comment:
Here we can add `array $value`

Reply:
No, it's possible just to pass string

Reply:
But even in your example in readme, you use an array:

```
whereGeoDistance('location', [-70, 40], 1000)
```

Reply:
Yes, but it's possible to pass string, I can add test for the case :) 

From docs:
```
"geo_distance" : {
                    "distance" : "12km",
                    "pin.location" : "40,-70"
                }

"geo_distance" : {
                    "distance" : "12km",
                    "pin.location" : "drm3btev3e86"
                }
```